tests/timeseries/_battery_pool/test_battery_pool.py

```diff
@@ -891,37 +925,68 @@ async def run_power_bounds_test(  # pylint: disable=too-many-locals
             {
                 "power_inclusion_lower_bound": -50,
                 "power_inclusion_upper_bound": math.nan,
+                "power_exclusion_lower_bound": -30,
+                "power_exclusion_upper_bound": 300,
             },
-            PowerMetrics(now, Bound(-60, 0), Bound(0, 9200)),
+            PowerMetrics(
+                now,
+                Bounds(Power.from_watts(-60), Power.from_watts(9200)),
+                Bounds(Power.from_watts(-600), Power.from_watts(500)),
+            ),
         ),
         Scenario(
             bat_inv_map[batteries_in_pool[0]],
             {
                 "active_power_inclusion_lower_bound": math.nan,
                 "active_power_inclusion_upper_bound": math.nan,
+                "active_power_exclusion_lower_bound": math.nan,
+                "active_power_exclusion_upper_bound": math.nan,
             },
-            PowerMetrics(now, Bound(-60, 0), Bound(0, 200)),
+            PowerMetrics(
+                now,
+                Bounds(Power.from_watts(-60), Power.from_watts(200)),
+                Bounds(Power.from_watts(-230), Power.from_watts(500)),
+            ),
         ),
         Scenario(
             batteries_in_pool[0],
-            {"power_inclusion_lower_bound": math.nan},
-            PowerMetrics(now, Bound(-10, 0), Bound(0, 200)),
+            {
+                "power_inclusion_lower_bound": math.nan,
+                "power_exclusion_lower_bound": math.nan,
+            },
+            PowerMetrics(
+                now,
+                Bounds(Power.from_watts(-10), Power.from_watts(200)),
+                Bounds(Power.from_watts(-200), Power.from_watts(500)),
+            ),
         ),
         Scenario(
             batteries_in_pool[1],
             {
                 "power_inclusion_lower_bound": -100,
                 "power_inclusion_upper_bound": math.nan,
+                "power_exclusion_lower_bound": -50,
+                "power_exclusion_upper_bound": 50,
             },
-            PowerMetrics(now, Bound(-100, 0), Bound(0, 6000)),
+            PowerMetrics(
+                now,
+                Bounds(Power.from_watts(-100), Power.from_watts(6000)),
+                Bounds(Power.from_watts(-200), Power.from_watts(500)),
+            ),
         ),
         Scenario(
             bat_inv_map[batteries_in_pool[1]],
             {
                 "active_power_inclusion_lower_bound": math.nan,
                 "active_power_inclusion_upper_bound": math.nan,
+                "active_power_exclusion_lower_bound": math.nan,
+                "active_power_exclusion_upper_bound": math.nan,
             },
-            PowerMetrics(now, Bound(-100, 0), Bound(0, 0)),
+            PowerMetrics(
+                now,
+                Bounds(Power.from_watts(-100), Power.from_watts(0)),
```

Comment:
Not super important but this could be:
```suggestion
                Bounds(Power.from_watts(-100), Power.zero()),
```